bottlecap

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+
+# bottlecap, 'cause it's kinda like cork... get it?
+
+usage() {
+	echo "usage:"
+	echo "bottlecap [--dev] [--runtime $runtime] [--build-dir $build_dir]  [--container $container] assemblerargs..."
```

Comment:
did you mean to evaluate the above variables when we print stuff to the screen? If not then we should use single quotes and not double quotes here.

Reply:
I do. You can do `bottlecap --build-dir foobar --help` and see your arg reflected in the help text, which I am a fan of personally.

Reply:
I see, I haven't seen that very much in practice, but seems OK